.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+# main is dev integration branch, merge main into release branch to trigger ga release.
+name: Release
+
+on:
+    workflow_dispatch:
+
+jobs:
+    lint-build-test:
+        name: Lint, Build, Test - Node ${{ matrix.node-version }} on ${{ matrix.os }}
+
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                node-version:
+                    - 16.x
```

Comment:
We only support LTS versions of Node.js. We should only have 18 and 20 in here.

Reply:
Thanks for pointing this out. version 16 is removed.